ld/armv7a7-imx6ull.ldt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Linker Template and Platform Config for ARMv7 A7 i.MX 6ULL
+ *
+ * Copyright 2021-2022 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef ARMV7A7_IMX6ULL_LDT
+#define ARMV7A7_IMX6ULL_LDT
+
+
+/* Platform specific definitions */
+#define SIZE_PAGE    0x1000
+#define SIZE_STACK   (2 * SIZE_PAGE)
+#define SIZE_HEAP    (2 * SIZE_PAGE)
+
+#define ADDR_DDR 0x80000000
+#define SIZE_DDR (128 * 1024 * 1024)
+
+#define ADDR_OCRAM 0x907000
+#define SIZE_OCRAM 0x11000
```

Comment:
This value is probably to less.
Linker error:
```
LD  plo-armv7a7-imx6ull.elf
LD  plo-ram-armv7a7-imx6ull.elf
arm-phoenix-ld: /tmp/plo-test/_build/armv7a7-imx6ull/prog/plo-ram-armv7a7-imx6ull.elf section `.bss' will not fit in region `m_ocram'
arm-phoenix-ld: /tmp/plo-test/_build/armv7a7-imx6ull/prog/plo-armv7a7-imx6ull.elf section `.bss' will not fit in region `m_ocram'
arm-phoenix-ld: region `m_ocram' overflowed by 36864 bytes
arm-phoenix-ld: region `m_ocram' overflowed by 36864 bytes
```
Curently it is [link](https://github.com/phoenix-rtos/plo/blob/76b9b92d6adbf734882ffbf7cd7b37e573006720/hal/armv7a/imx6ull/config.h#L45)
```
#define SIZE_OCRAM 0x020000
```

Reply:
That's actually a very good sign, and a pro for the addition of linker scripts, previously it was not detected. Actually the size of OCRAM available to `plo` is 68 KB. This was discussed in the https://github.com/phoenix-rtos/plo/pull/167#discussion_r829001059 above (which is now marked as resolved).

I need to investigate this, previously it was (0x20000) 128KB (wrong assumption due to 68KB available), now maybe we just don't fit in this area and this is now being detected by the linker after adding the linker scripts.